PurchasesCoreSwiftTests/FoundationExtensions/DictionaryExtensionsTests.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  NSDictionaryExtensionsTests.swift
+//  PurchasesCoreSwiftTests
+//
+//  Created by Andrés Boedo on 9/28/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import XCTest
+import Nimble
+
+@testable import PurchasesCoreSwift
+
+class DictionaryExtensionsTests: XCTestCase {
+    
+    func testRemovingNSNullValuesFiltersCorrectly() {
+        let testValues: [String : Any] = [
+            "instrument": "guitar",
+            "type": 1,
+            "volume": NSNull()
+        ]
+        let expectedValues: [String : Any] = [
+            "instrument": "guitar",
+            "type": 1
+        ]
+        let removed = testValues.removingNSNullValues()
```

Comment:
being picky here, but the naming `removed` makes me think that the result of this method is the removed values. can we rename to something else, maybe `nonNullDict` or something?

Reply:
well, this code was there, I mean, it doesn't belong to this PR. That said, I don't mind change it 🙂 Let me know 🤙

Reply:
if you don't mind while you're in it :)

Reply:
What do you think about `obtainedValues`? I usually use the same structure in the tests. I defined an `expected` var and an `obtained` var, then I compare both the expected and the obtained value.